test/ci/simple-regional-with-networking.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Files under `test/ci` are obsolete and will be removed soon. Please follow the pattern in `build/int.cloudbuild.yaml` to define create, converge, verify, and destroy steps for the `simple-regional-with-networking-local` Kitchen instance.

Reply:
Done.